src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -293,10 +321,10 @@ private void validateSequenceDictionaries() {
         //check reference dict against reads dict
         if (hasReference() && hasReads()) {
             if (hasIntervals()){
-                SequenceDictionaryUtils.validateDictionaries("reference", refDict, "reads", readDict, true, intervalsForTraversal);
+                SequenceDictionaryUtils.validateDictionaries("reads", readDict, "reference", refDict, true, intervalsForTraversal);
```

Comment:
I think the reference dictionary was better as the first parameter -- it seems more logical to say that the first thing is a superset of the second, rather than the other way around.